docs/build/guides/conventions/workspace.mdx

```diff
@@ -0,0 +1,357 @@
+---
+title: Workspaces
+hide_table_of_contents: true
+description: Organize contracts using Cargo workspaces
+---
+
+### Initializing workspace project
+
+Using Cargo's workspace [feature](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html) makes it very convenient to organize your smart contracts in subdirectories of your project's root.
+
+It's very simple to get started using the cli:
+
+```
+stellar contract init . --name add_contract
+```
+
+Running this command will create a root project directory (`soroban-project`) and then initialize a project workspace with a single contract named `add_contract`.
+
+Adding one more contract template to the project can be done using the same command:
+
+```
+stellar contract init . --name main_contract
+```
+
+The project tree will look like this:
+
+```
+$ tree
+.
+├── Cargo.toml
+├── contracts
+│   ├── add-contract
+│   │   ├── Cargo.toml
+│   │   ├── Makefile
+│   │   └── src
+│   │       ├── lib.rs
+│   │       └── test.rs
+│   └── main-contract
+│       ├── Cargo.toml
+│       ├── Makefile
+│       └── src
+│           ├── lib.rs
+│           └── test.rs
+└── README.md
+```
+
+Running `stellar contract init` command created a sample contracts subdirectories (located in `./contracts), each contains:
+
+1. Cargo.toml file with all (minimal) necessary dependencies
+2. `Makefile` to easily build contract using `make` command
+3. `src` directory with an example hello-world contract and a simple test.
+
+We can now build our workspace's contracts with
+
+```
+stellar contract build
+```
+
+command and check the build directory `target/wasm32-unknown-unknown/release/`
```

Comment:
I think we can structure this without breaking the sentence across paragraphs.
````suggestion
Build the contracts with the following command, and check the build directory `target/wasm32-unknown-unknown/release/` for the compiled `.wasm` files.

```
stellar contract build
```
````